Rellenar.py

```diff
@@ -1,30 +1,27 @@
 scriptPath = nuke.root().knob('name').value()
-scriptName = ((scriptPath.split('/'))[(len((scriptPath.split('/')))-1)]).split('.')[0]
+scriptName = scriptPath.split('/')[-1].split('.')[0]
 
-cap = scriptName.split('_')[0]
-sec = scriptName.split('_')[1]
-plano = scriptName.split('_')[2]
-dep = scriptName.split('_')[3]
-op = scriptName.split('_')[4]
-vCliente = (scriptName.split('_')[5])[1]
-vOp = ((scriptName.split('_')[5])[2]) + ((scriptName.split('_')[5])[3])
+[cap, sec, plano, dep, op, version] = scriptName.split('_')
+vCliente = version[1]
+vCliente = scriptNameSections[5][1]

```

Comment:
Me da error aquí, creo que porque no hemos definido la variable scriptNamesSections. Pero quitando esta parte del código todo funciona bien, de hecho uno de los cambios que le tengo que hacer al AutoWrite para esta nueva temporada pasa por quitar la diferencia entre versión de cliente y de operador, con lo que va a ser más fácil todo

Reply:
Correcto, esa se nos pasó. Para este tipo de fallos hay un tipo de programas que se llaman "linters", que te marcan fallos de este tipo como que una variable no está definida o que una variable no se está usando. Yo tengo instalado uno en VSCode que se llama ESLint. Instálala en el VSCode dándole al icono de los cuadraditos en la barra de la izq y buscando este.
![image](https://user-images.githubusercontent.com/17493200/98342735-2576af00-2011-11eb-935f-4503323afae6.png)

Me imagino que `scriptNameSections` era `scriptName.split('_')`, que ya lo habíamos descompuesto, en la línea 4 (`[cap, sec, plano, dep, op, version] = scriptName.split('_')`).

Puede que sólo haya que hacer este cambio
```suggestion
vCliente = version[1]
```

Si crees que ese es el cambio que hace falta, me parece que te sale un botón con el que puedes hacer el cambio directamente desde mi comentario. Si no lo ves claro esta tarde lo hablamos.

Estaría bien que te hiciese una mini-guía de git, y tb quería hacerla para 2 amigos que están aprendiendo a programar, así que probablemente escriba algo este finde para que le puedas sacar más partido a git.